run-make-check.sh

```diff
@@ -55,6 +55,8 @@ function detect_ceph_dev_pkgs() {
     local boost_root=/opt/ceph
     if test -f $boost_root/include/boost/config.hpp; then
         cmake_opts+=" -DWITH_SYSTEM_BOOST=ON -DBOOST_ROOT=$boost_root"
+    else
+        cmake_opts+=" -DBOOST_J=$(get_processors)"
```

Comment:
get_processors is nproc / 2 so this patch reduce the number of used processors

Reply:
why do you think "this patch reduce the number of used processors"? see https://github.com/ceph/ceph-build/blob/master/ceph-pull-requests/config/definitions/ceph-pull-requests.yml#L56

Reply:
and what change you are requesting?

Reply:
Ok. By default get_processors is nproc / 2 but we override nproc, so you are right, this is equal.
maybe get_processors could be nproc so we don't have to override that default behavior we don't use

Reply:
i'd prefer not, please see https://github.com/ceph/ceph-build/pull/1103 .